packages/app/obojobo-express/config.js

```diff
@@ -24,30 +24,47 @@ if (process.env.DATABASE_URL) {
 	process.env.DB_PORT = dburl.port
 }
 
-const replaceENVsInObject = configObject => {
-	const rawJson = JSON.stringify(configObject) // convert back to string
+const isStringJSON = (name, string) => {
+	try{
```

Comment:
Regarding comment `"avoid problems in assuming values that start with { or [ are supposed to be json"` This automatically parses string as JSON here